### PR TITLE
[+] FO : the cart summary is updated according to delivery costs

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: themes/StarterTheme/_dev/js/checkout.js [hunk 1]
```diff
@@ -90,12 +96,23 @@ function setupCheckoutScripts () {
...
+
     $.get('', {
       action: 'getCartSummary'
     }).then(resp => {
       $('#cart-summary').html(resp);
     });
   });
+
+  prestashop.on('cart updated', function () {
+    // First perform the action using AJAX
```
Comment (djfm) [inline themes/StarterTheme/_dev/js/checkout.js]: copy/pasted comment is not valid here :)

File: themes/StarterTheme/_dev/js/checkout.js [hunk 2]
```diff
@@ -90,12 +96,23 @@ function setupCheckoutScripts () {
   collapsePaymentOptions();
 
   prestashop.on('cart updated', function () {
+    if ($('body#order-opc').length == 0) {
```
Comment (djfm) [inline themes/StarterTheme/_dev/js/checkout.js]: where do we have a body#order-opc?

File: themes/StarterTheme/_dev/js/checkout.js [hunk 3]
```diff
@@ -90,12 +96,23 @@ function setupCheckoutScripts () {
   collapsePaymentOptions();
 
   prestashop.on('cart updated', function () {
```
Comment (djfm) [inline themes/StarterTheme/_dev/js/checkout.js]: Not sure this handler is still useful, is it?

File: themes/StarterTheme/_dev/js/checkout.js (hunk 2 above)
Comment (mickaelandrieu) [inline themes/StarterTheme/_dev/js/checkout.js]: nowhere, this is the proof that this trigger is totaly useless, I remove it.

File: themes/StarterTheme/_dev/js/checkout.js (hunk 3 above)
Comment (mickaelandrieu) [inline themes/StarterTheme/_dev/js/checkout.js]: yeah, you confirm my doubts. Let's the tests ensure we have right.

Comment (djfm) [inline themes/StarterTheme/_dev/js/checkout.js]: tests are fine without this

**Comment by mickaelandrieu** [comment]
thansk for review @djfm

**Pull request merged**